src/ai_marketplace_monitor/telegram.py

```
@@ -0,0 +1,152 @@
+            return
+        if not isinstance(self.telegram_chat_id, str) or not self.telegram_chat_id:
+            raise ValueError("An non-empty telegram_chat_id is needed.")
+        self.telegram_chat_id = self.telegram_chat_id.strip()
+
+    def handle_message_format(self: "TelegramNotificationConfig") -> None:
+        super().handle_message_format()
+        # Override default to markdown for Telegram
+        if self.message_format == "plain_text":
+            self.message_format = "markdown"
+
+    def send_message(
```

Pylint (3.3.7)

[refactor] Too many local variables (26/15)

(R0914)